lib/os/CMakeLists.txt

```
@@ -25,6 +23,14 @@ zephyr_sources(
   multi_heap.c
   )
 
+zephyr_sources_ifdef(CONFIG_ONOFF
+  onoff.c
+  )
+
+zephyr_sources_ifdef(CONFIG_NOTIFY
+    notify.c
+  )
```

inconsistent spacing. also could be one liners

```suggestion
zephyr_sources_ifdef(CONFIG_ONOFF onoff.c)
zephyr_sources_ifdef(CONFIG_NOTIFY notify.c)
```